R/zchunk_L111.nonghg_en_R_S_T_Y.R

```diff
@@ -1,110 +1,303 @@
+
+    # These are provided in the EDGAR inventory at the global level only.
+    L111.nonghg_tg_R_en_Si_F_Yh_all.tmp1 %>%
+      filter(EDGAR_agg_sector %in% c("trn_intl_ship", "trn_intl_air")) ->
+      L111.nonghg_tg_R_en_Si_F_Yh_intl.tmp1
+
+    L111.nonghg_tg_R_en_Si_F_Yh_all.tmp1 %>%
+      filter(!(EDGAR_agg_sector %in% c("trn_intl_ship", "trn_intl_air"))) ->
+      L111.nonghg_tg_R_en_Si_F_Yh_dom.tmp1
+
+    # Calculate total energy for international aviation emissions.
+    L111.nonghg_tg_R_en_Si_F_Yh_intl.tmp1 %>%
+      group_by(EDGAR_agg_sector, Non.CO2, year) %>%
+      summarize(energy = sum(energy))->
+      L111.en_Si_F_Yh_intl.tmp1
+
+    # Compute international input emissions based on total emission and total emission shares.
+    L111.nonghg_tg_R_en_Si_F_Yh_intl.tmp1 %>%
+      left_join(L111.en_Si_F_Yh_intl.tmp1, by = c("EDGAR_agg_sector", "Non.CO2", "year")) %>%
+      rename(tot_energy = energy.y, energy = energy.x) %>%
+      left_join(L111_EDGAR_intl, by = c("EDGAR_agg_sector", "Non.CO2", "year")) %>%
+      mutate(tot_emiss = value / 1000.0)  %>%
+      mutate(energy_share = energy / tot_energy) %>%
+      mutate(emissions = tot_emiss * energy_share)  %>%
+      # Remove unnecessary columns and rows.
+      select(-tot_energy, -tot_emiss, -energy_share, -value) %>%
+      na.omit()  %>%
+      # Add domestic domestic emissions back in.
+      bind_rows(L111.nonghg_tg_R_en_Si_F_Yh_dom.tmp1) ->
+      L111.nonghg_tg_R_en_Si_F_Yh.tmp2
+
+    # Compute non-ghg emission totals by GCAM sector, fuel, technology, and driver type for EDGAR historical years.
+
+    L111.nonghg_tg_R_en_Si_F_Yh.tmp2 %>%
+      left_join_keep_first_only(GCAM_sector_tech, by = c("sector", "fuel", "technology")) %>%
+      group_by(GCAM_region_ID, Non.CO2, supplysector, subsector, stub.technology, year) %>%
+      summarize(input.emissions = sum(emissions)) %>%
+      filter(year, year %in% emissions.EDGAR_YEARS) ->
+      L111.nonghg_tg_R_en_S_F_Yh
+
+    # Compute non-ghg emission factors for GCAM sector, fuel, technology, and driver type for EDGAR historical years.
+
+    # Calculate energy totals by region, sector, and fuel.
+    L101.in_EJ_R_en_Si_F_Yh %>%
+      left_join(GCAM_sector_tech, by = c("sector", "fuel", "technology")) %>%
+      # Remove unneeded columns.
+      select(GCAM_region_ID, sector, fuel, technology, year, value, supplysector, subsector, stub.technology) %>%
+      group_by(GCAM_region_ID, supplysector, subsector, stub.technology, year) %>%
+      summarize(energy = sum(value)) ->
+      L101.in_EJ_R_en_S_F_Yh.tmp
+
+    # Calculate emission shares by energy totals.
+    L111.nonghg_tg_R_en_S_F_Yh %>%
+      left_join(L101.in_EJ_R_en_S_F_Yh.tmp, by = c("GCAM_region_ID", "supplysector", "subsector", "stub.technology", "year")) %>%
+      mutate(emfact = input.emissions / energy) %>%
+      mutate_all(funs(replace(., is.na(.), 0))) %>%
+      mutate(emfact = replace(emfact, emfact == Inf, 0)) ->
```

Comment:
The downstream chunks are going to be expecting `value` not `emfact` I suspect. Either change it here, or change downstream.